tests/TestCase.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace PodPoint\Reviews\Tests;
+
+use Mockery;
+use PodPoint\Reviews\AbstractApiClient;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\StreamInterface;
+
+class TestCase extends \PHPUnit\Framework\TestCase
```

Comment:
Function description in the docblocks for the three functions in this file please :pray: